frontend/packages/dev-console/src/components/projects/details/__tests__/ProjectDetailsPage.spec.tsx

```diff
@@ -4,23 +4,28 @@ import { Redirect } from 'react-router';
 import { ALL_NAMESPACES_KEY } from '@console/shared';
 import { BreadCrumbs } from '@console/internal/components/utils';
 import { ProjectDetailsPage } from '../ProjectDetailsPage';
+import CreateProjectListPage from '../../CreateProjectListPage';
+
+const match = { url: '', params: {}, isExact: true, path: '' };
 
 describe('ProjectDetailsPage', () => {
-  it('expect ProjectDetailsPage to render a redirect when in the all-projects namespace', () => {
-    const component = shallow(<ProjectDetailsPage activeNamespace={ALL_NAMESPACES_KEY} />);
+  it('expect ProjectDetailsPage to render the create project page when in the all-projects namespace', () => {
+    const component = shallow(
```

Comment:
too generic name for the wrapper

Reply:
I'm unclear as to what is too generic. Could you suggest?

Reply:
Ya unsure on this comment. A lot of tests use `wrapper` but that's really no better. I'm fine with whatever is used for the name here.